source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,128 @@
 Resolver for Home Networks
+==========================
+
```

Comment:
Perhaps an intro on why you would want to do this? Perhaps also a good place to debunk some myths on privacy and security coming from the people who make the software.

Reply:
I agree. Extra sections for configuring DoT/DoH and/or RPZ could also be for the home user.
Although information on where to get those resources should be linked externally as to avoid bias.
For DoT/DoH: https://dnsprivacy.org/wiki/display/DP/DNS+Privacy+Public+Resolvers
For RPZ there is no such link and most RPZ feeds are from commercial services. If you write such a section ping me to see what I can come up with.